### ISSUE TITLE
Breaking change in 0.10.5

### ISSUE THREAD

**Issue by erdnaxeli** [theskumar/python-dotenv#233]
The release 0.10.5 breaks some invalid case parsing.

In .env:
```
LOL='mdr','xd'
```

In toto.py:
```
import os
from dotenv import load_dotenv

load_dotenv()
print(os.environ['LOL'])
```

Version 0.10.3:
```
$ poetry run python toto.py
mdr
```

Version 0.10.4
```
$ poetry run python toto.py
Python-dotenv could not parse statement starting at line 1
mdr
```

Version 0.10.5
```
$ poetry run python toto.py
Python-dotenv could not parse statement starting at line 1
Traceback (most recent call last):
  File "toto.py", line 5, in <module>
    print(os.environ['LOL'])
  File "/home/…/.pyenv/versions/3.7.3/lib/python3.7/os.py", line 678, in __getitem__
    raise KeyError(key) from None
KeyError: 'LOL'
```

I know it was a bug before, or the string was not correctly parsed, and the behavior it has now is the expected one. But it worked before, even in a wrong way, and now it breaks. I know too this is a pre-release, not a stable one, as a majority of python lib, but still some people use it everyday in production.

I don't know what to propose to avoid this situation in the future. The release number could have been bumped to 0.11.0, but as I actually use the environs which does not lock the version of python-dotenv, the result would have been the same.

**Comment by bbc2** [comment]
Thank you for the feedback.  I understand the frustration.  I guess 0.10.5 should have been released as 0.11.0 but I didn't expect cases such as yours to break.

I would like to make this package stable (1.x.y) but we're not there yet: there are still quite a few issues to fix.  I hope that the future breaking changes will be minor.

About your case: I think another valid behavior would have been to map `LOL` to `mdr,xd`, as Bash would have done it, but I'm not sure we should allow that syntax since it's more complex than necessary (`LOL='mdr,xd'` would achieve the same result in a simpler way).

**Comment by erdnaxeli** [comment]
Thanks for reading my feedback :)

Let's hope this situation does not happens again. Getting to 1.x would be a big step inded!

I will close this issue now, there isn't much to do.
